src/brainbrowser/volume-viewer/volume-loaders/nifti1.js

```diff
@@ -392,22 +392,45 @@
     }
 
     var d = 0;                  // Generic loop counter.
-
-    // Convert data to bytes.
-    byte_data = new Uint8Array(native_data.length);
-    n_min = native_data[0];
-    n_max = native_data[0];
-
-    for (d = 1; d < native_data.length; d++) {
-      if (native_data[d] > n_max)
-        n_max = native_data[d];
-      if (native_data[d] < n_min)
-        n_min = native_data[d];
+    var value = 0;              // Scaled value.
+    var slope = header.scl_slope;
+    var inter = header.scl_inter;
+    var n_min, n_max;
```

Comment:
@rdvincent  You can also avoid to have two times the same code for the min and max search if you redefine native_data[d], before the min and max search. It's hope to you to changed it or not. I will merged it tomorrow.